diagnostics-app/package.json

```diff
@@ -25,6 +41,20 @@
     "tslib": "^2.1.0",
     "zone.js": "~0.11.4"
   },
+  "//devDependencies": {
+    "@angular-devkit/build-angular": "This package contains Architect builders used to build and test Angular applications and libraries",
+    "@angular/cli": "Provides a CLI tool that is used to scaffold and maintain Angular applications",
+    "@angular/compiler-cli": "Used for ahead of time compilation",
+    "@types/jasmine": "Typescript types for Jasmine",
+    "@types/node": "Typescript types for NodeJS",
+    "jasmine-core": "Testing library",
+    "karma": "Test runner",
+    "karma-chrome-launcher": "Used for E2E Testing",
+    "karma-coverage": "Coverage reports for Karma.",
```

Comment:
nit: please be consistent with dots at the end of comments.

Reply:
Thanks for pointing out!